### ISSUE TITLE
DeprecationWarning for inspect.getargspec()

### ISSUE THREAD

**Issue by The-Compiler** [pytest-dev/pluggy#105]
On Python 3.6:

```
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/_pytest/pytester.py", line 738, in runpytest_inprocess
    reprec = self.inline_run(*args, **kwargs)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/_pytest/pytester.py", line 711, in inline_run
    ret = pytest.main(list(args), plugins=plugins)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/_pytest/config.py", line 50, in main
    config = _prepareconfig(args, plugins)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/_pytest/config.py", line 156, in _prepareconfig
    pluginmanager.register(plugin)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/_pytest/config.py", line 245, in register
    ret = super(PytestPluginManager, self).register(plugin, name)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/pluggy/__init__.py", line 265, in register
    hookimpl = HookImpl(plugin, plugin_name, method, hookimpl_opts)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/pluggy/__init__.py", line 721, in __init__
    self.argnames, self.kwargnames = varnames(self.function)
  File "/home/florian/proj/qutebrowser/git/.tox/py36/lib/python3.6/site-packages/pluggy/__init__.py", line 561, in varnames
    spec = inspect.getargspec(func)
  File "/usr/lib64/python3.6/inspect.py", line 1068, in getargspec
    DeprecationWarning, stacklevel=2)
DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()
```

**Comment by tgoodlet** [comment]
Addressed by #81.
Still waiting on the preferred `funcsigs` fix however.

**Comment by tgoodlet** [comment]
@The-Compiler fix is in [`0.6.0`](https://pypi.org/project/pluggy/)

**Comment by nicoddemus** [comment]
Thanks! Updated the pin in the `release-3.3.0` branch.
